### ISSUE TITLE
Misspelling of cmdlet New-TenantAllowBlockListItem in a few places

### ISSUE THREAD

**Issue by rogermicrosoft** [MicrosoftDocs/microsoft-365-docs#4325]
Hi!
At a couple of places the cmdlet "New-TenantAllowBlockListItems" is spelled without the s in the end New-TenantAllowBlockListItem :-)!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 94b60171-271c-fb93-88bf-0cd9db014541
* Version Independent ID: e9cd207e-0d4c-e268-dd3a-701e1e2df658
* Content: [Manage your allows and blocks in the Tenant Allow/Block List - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/tenant-allow-block-list?view=o365-worldwide#feedback)
* Content Source: [microsoft-365/security/office-365-security/tenant-allow-block-list.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/tenant-allow-block-list.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@rogermicrosoft, fixed. In the future, you too can make changes like this directly in the topic (no need to open a GitHub Issue first). For instructions, see <https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/README.md>.

Also note, that typo came from the examples in the [New-TenantAllowBlockListItems](https://docs.microsoft.com/powershell/module/exchange/new-tenantallowblocklistitems) cmdlet reference topic itself, so I fixed them there, too.

Thanks for the feedback.

Closing
